SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -11,14 +11,15 @@
 Summary:        Mariner kernel that has MSHV Host support
 Name:           kernel-mshv
 Version:        5.15.98.mshv1
-Release:        2%{?dist}
+Release:        3%{?dist}
 License:        GPLv2
-URL:            https://microsoft.visualstudio.com/DefaultCollection/LSG/_git/linux-dom0
+URL:            https://github.com/microsoft/CBL-Mariner-Linux-Kernel
```

Comment:
Wondering if this should be reverted back based on this [feedback](https://github.com/microsoft/CBL-Mariner/pull/5147#discussion_r1149292233)

Reply:
CBL-Mariner-Linux-Kernel URL is definitely not right :) Let's follow up with the right people to give advice in such situations.